### ISSUE TITLE
Won't load

### ISSUE THREAD

**Issue by GoogleCodeExporter** [MziRintu/iphonebrowser#79]
```
What steps will reproduce the problem?
1. I double click to open iPhone Browser and I get an error (
AMDeviceNotificationSubscribe failed with error -402653085)
2.
3.

What is the expected output? What do you see instead?
I expect so see the opening screen of iPhone Browser.  It shuts down instead.


What version of the product are you using? On what operating system?
Latest version on Vista

Please provide any additional information below.


```

Original issue reported on code.google.com by `justinjs...@gmail.com` on 4 Apr 2009 at 9:37

**Comment by GoogleCodeExporter** [comment]
```
I had the same error.
Try starting/restarting the "Apple Mobile Device" windows service.
Let me know ... ;-)
```

Original comment by `manuel.p...@gmail.com` on 19 May 2009 at 1:31

**Comment by GoogleCodeExporter** [comment]
```
Please try with v1.90 - do you have iTunes 8.2 installed?
```

Original comment by `NetMage....@gmail.com` on 12 Jun 2009 at 8:24
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Please try v1.91 - this version works with Windows 7 and hopefully Vista
```

Original comment by `NetMage....@gmail.com` on 18 Jun 2009 at 6:12

**Comment by GoogleCodeExporter** [comment]
Original comment by `NetMage....@gmail.com` on 18 Jun 2009 at 6:12
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
@manuel.puricelli

did what you suggested, solved my problem! I am running iPhoneBrowser v1.9.3 on 
Win7.

Thanks alot!
```

Original comment by `bwdub...@gmail.com` on 18 Feb 2010 at 8:24
